Kickstarter-iOS/AppDelegate.swift

```diff
@@ -238,11 +236,17 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
     open url: URL,
     options: [UIApplication.OpenURLOptionsKey: Any] = [:]
   ) -> Bool {
-    return self.viewModel.inputs.applicationOpenUrl(
-      application: app,
-      url: url,
-      options: options
-    )
+    let fbReturnValue = ApplicationDelegate.shared.application(app, open: url, options: options)
+
+    if !fbReturnValue {
```

Comment:
If FB won't handle the URL, we just return whether we handle it which is always `true`.